config/generateRunFile.coffee

```diff
@@ -659,6 +777,9 @@ generateDev = (KONFIG, options) ->
     elif [ "$1" == "switchclient" ]; then
       switch_client_version $2
 
+    elif [ "$1" == "k8s" ]; then
+      k8s $2
```

Comment:
For forward-compatibility, you can shift the first parameter and pass the rest to the `k8s` function. E.g.:

```shell
...
elif [ "$1" == "k8s" ]; then
  shift
  k8s "$@"
...
```